plutus-chain-index/README_alpha-release_Marconi.md

```diff
@@ -0,0 +1,78 @@
+
+Marconi gives you the ability to customize the set of data that is indexed. It provides a library of functionality with which you can do the following:
+* build your own indexers, 
+* work only with the data that you care about for your application, 
+* use whatever storage engine you prefer, and 
+* support only the queries that you need to support.
+
+### Scalability through Customized Solutions for Each Use Case
+
+The philosophy behind Marconi is to maintain scalability by customizing the solution for each use case. You can filter the streams of information to transmit only the data you need for your dApp, minimizing network traffic and database storage. You can also customize database schemas and queries so that the dApps have the most efficient and scalable application-specific API to interact with the data.
+
+### Indexing solution ### 
+
+Any indexing solution must deal with blockchain rollbacks, which can happen as the network achieves consensus. Marconi is designed to deal with rollbacks as elegantly as possible while simultaneously finding an effective balance between memory, disk and CPU usage. 
+
+Managing rollbacks is very simple and fast. Marconi drops the events that were rolled back. Because the events that can be rolled back are stored in memory, no complicated logic is required to undo the projection of the list of events on disk, which Marconi would need if it stored everything on disk as quickly as possible. 
+
+Making `K` configurable makes the design quite scalable. Developers do not usually need to guard themselves against rollbacks by `K` blocks, so they can choose to store only 10 events in memory, for example, thereby sacrificing the correctness of handling rollbacks for memory usage.
+
+In the event of a restart, recovery is very simple. Marconi stores only fully confirmed transactions, so there is nothing to do other than to load the state persisted on disk and to follow the blockchain from the corresponding chain-point. 
+
+## Queries
+
+The indexed data is accessible through queries. There are no constraints on the format of queries or on the results. Both are identified by a type variable that the indexer exposes. The user, such as a dApp developer, provides the code for the query, result types and the functions used to query and store events on disk. 
+
+This is currently the only area where you can customize the indexer by providing functions that handle queries and storage. Note that the queries have to run on both the in-memory data and the data that is stored.
+
+# Installation
+
+Prerequisite: You already have a Cardano node running. 
+
+## Example command for launching Cardano node 
+
```

Comment:
```suggestion

The synchronisation should take about 3.5h and it will consume around 9Gb of RAM, when synchronising both indexers at the same time.

```